research-hub-web/src/app/app.component.html

```diff
@@ -1,4 +1,8 @@
-<section *ngIf="viewIsLoaded" class="mat-typography">
+<app-layout>
+  <router-outlet></router-outlet>
+</app-layout>
+
+<section *ngIf="false" class="mat-typography">
```

Comment:
Just a note: this entire section won't be rendered. I've only left it in to have easy access to the code while I'm refactoring. You can safely ignore all the changes here, as the whole section will ultimately be removed